lib/__tests__/index-test.js

```diff
@@ -667,14 +689,12 @@ describe('device is set to mobile', () => {
 });
 
 describe('exclusions', () => {
-  var createElement = React.createElement;
-
   before(() => {
     a11y(React, { exclude: ['REDUNDANT_ALT'] });
   });
 
   after(() => {
-    React.createElement = createElement;
+    a11y.restore(React);
```

Comment:
@ljharb the `restore` method added to `after` doesn't take an argument, but the `restore` method added to `a11y` does. This illustrates a good point that I should give them different names